.github/changelog-configuration.json

```diff
@@ -0,0 +1,20 @@
+{
+    "categories": [
```

Comment:
The config allows setting some formatting option per PR.
Default config looks like this:

![image](https://user-images.githubusercontent.com/12159026/103295293-eef75800-49f3-11eb-8b9b-50cf14267c09.png)

Maybe put the PR number onto the same single line?
```suggestion
    "pr_template": "- ${{TITLE}} (PR #${{NUMBER}})",
    "categories": [
```

Reply:
> Maybe put the PR number onto the same single line?

Yes, please :+1: